Lib/test/libregrtest/logger.py

```diff
@@ -84,3 +176,166 @@ def stop_load_tracker(self) -> None:
             return
         self.win_load_tracker.close()
         self.win_load_tracker = None
+
+
+class StatusRemovingWrapper(io.TextIOWrapper):
+    # Set status_size here to avoid having to override __init__.
+    status_size = 0
+    def write(self, msg):
```

Comment:
```suggestion

    def write(self, msg: str) -> None:
```

https://github.com/python/typeshed/blob/f7aa7b709a826ed34f52b1de3f7095f90f349a9c/stdlib/io.pyi#L143